src/iqm/cortex_cli/cortex_cli.py

```diff
@@ -134,6 +137,20 @@ def _validate_path(ctx: click.Context, param: click.Path, path: str) -> str:
         return path
     ctx.obj[param.name] = True
 
+    # Skip parameter validation completely if the parameter comes from command line
+    param_name = param.name
+    if isinstance(param_name, str):
+        param_source = ctx.get_parameter_source(param_name)
+        if isinstance(param_source, click.core.ParameterSource):
+            if param_source.name == 'COMMANDLINE':
+                cmd_line_opt_name = param.opts[0]
+                msg = click.style(
+                    f'Skipping validation of "{cmd_line_opt_name}", using the provided value "{path}" as is',
+                    fg='yellow',
+                )
+                click.echo(msg)
+                return path
```

Comment:
This check happens three times, could be a function.

Reply:
Refactored